eng/testing/xunit/vstest.props

```diff
@@ -2,19 +2,15 @@
   <PropertyGroup>
     <!-- Microsoft.Net.Test.Sdk brings a lot of satellite assemblies in. -->
     <SatelliteResourceLanguages>en</SatelliteResourceLanguages>
+    <TrxTestResultsName>testResults.trx</TrxTestResultsName>
```

Comment:
Do we know if the helix sdk supports .trx files to show the test results in AzDo, submitting to kusto and failing the build? If not we should request a feature for when we want to move to use this as our default test runner. cc: @alexperovich

Reply:
Helix supports trx files for uploading to azure devops. Kusto is only supported for xunit.

Reply:
I think a lot of people use Kusto for metrics and what not as the AzDo APIs aren't that great to get test results and search through them. Would it be possible to add Kusto from a trx file?

Reply:
The support for processing xunit results into kusto is on the server side for helix and extending that for other formats would be a very large an potentially disruptive change. The azure devops uploading code is in arcade and very easy to change so it supports many formats. My understanding is that we want to consolidate everything to azure devops and make that experience better, rather than continue investing in our custom reporting. @Chrisboh may have more context.

Reply:
Yeah I knew that trx is supported but I didn't know about Kusto only working with the xunit xml format.